packages/backend/jest.setup.js

```diff
@@ -76,67 +76,12 @@ module.exports = async (globalConfig) => {
 
   const setupTigerbeetle = async () => {
     if (!process.env.TIGERBEETLE_REPLICA_ADDRESSES) {
-      const { name: tigerbeetleDir } = tmp.dirSync({ unsafeCleanup: true })
-
-      const tbContFormat = await new GenericContainer(
-        'ghcr.io/coilhq/tigerbeetle@sha256:c312832a460e7374bcbd4bd4a5ae79b8762f73df6363c9c8106c76d864e21303'
-      )
-        .withExposedPorts(TIGERBEETLE_PORT)
-        .withBindMounts([
-          {
-            source: tigerbeetleDir,
-            target: TIGERBEETLE_DIR
-          }
-        ])
-        .withAddedCapabilities('IPC_LOCK')
-        .withCommand([
-          'init',
-          '--cluster=' + TIGERBEETLE_CLUSTER_ID,
-          '--replica=0',
-          '--directory=' + TIGERBEETLE_DIR
-        ])
-        .withWaitStrategy(Wait.forLogMessage(/initialized data file/))
-        .start()
-
-      const streamTbFormat = await tbContFormat.logs()
-      if (TIGERBEETLE_CONTAINER_LOG) {
-        streamTbFormat
-          .on('data', (line) => console.log(line))
-          .on('err', (line) => console.error(line))
-          .on('end', () => console.log('Stream closed for [tb-format]'))
-      }
-
-      // Give TB a chance to startup (no message currently to notify allocation is complete):
-      await new Promise((f) => setTimeout(f, 1000))
-
-      const tbContStart = await new GenericContainer(
-        'ghcr.io/coilhq/tigerbeetle:dj-request-dirty-prepare@sha256:c312832a460e7374bcbd4bd4a5ae79b8762f73df6363c9c8106c76d864e21303'
+      const tbContStart = await tigerbeetle.startTigerbeetleContainer(
+        TIGERBEETLE_DIR,
+        TIGERBEETLE_PORT,
+        TIGERBEETLE_CLUSTER_ID,
+        TIGERBEETLE_CONTAINER_LOG
```

Comment:
Should this also use values from `Config`?

Reply:
You mean importing that from `dist` too?

Reply:
Is there ever a case where we want to have different configs?

Reply:
As is, it seems like we'd be fine just using `Config` and not passing anything to `startTigerbeetleContainer`

Reply:
:thinking: or does that work if `jest.setup.ts` is defining environment variables used by `Config` (used by `startTigerbeetleContainer`)?

Reply:
I ended up only having those in Config that index.ts uses and having all others in `src/tests/tigerbeetle.ts`, i.e. where `startTigerbeetleContainer` is defined. What do you think?